.ci/ci_entry_point.sh

```diff
@@ -77,7 +77,8 @@ if [ "${repo_to_test}" == "${tests_repo}" ]; then
 		git fetch origin "pull/${pr_number}/head:${pr_branch}"
 		git checkout "${pr_branch}"
 		rebase_merge_flag="--rebase-merges"
-		if [[ ${ID} == "ubuntu" && ${VERSION_ID} == "18.04" ]]; then
+		has_rebase_merges=$(git rebase --help 2>&1|grep -- --rebase-merges || true)
+		if [[ -n "${has_rebase_merges}" ]]; then
```

Comment:
- Nit: Only one set of square brackets is required here.
- Don't you mean `[ -z "${has_rebase_merges}" ] && rebase_merge_flag="--preserve-merges"` ?

Reply:
Yes, I definitely meant `-z` rather than `-n`! I'd love to blame Monday morning, but it's not the first time my brain has got those two muddled up! Should be fixed now, thanks for the correction!